patches/nuget-client/0005-Remove-unneeded-feeds.patch

```diff
@@ -1,24 +1,24 @@
-From 79dd89633d9975d5987b7722591923d543d26181 Mon Sep 17 00:00:00 2001
+From b899061f2875d9885510db141f0ffc4e0d92edba Mon Sep 17 00:00:00 2001
 From: adaggarwal <aditya.aggarwal@microsoft.com>
 Date: Tue, 29 Oct 2019 17:38:08 +0000
-Subject: [PATCH] Remove-unneeded-feeds
+Subject: [PATCH 5/7] Remove-unneeded-feeds
 
 ---
  NuGet.Config | 1 -
  1 file changed, 1 deletion(-)
 
 diff --git a/NuGet.Config b/NuGet.Config
-index 440c8b5..1c2ddda 100644
+index fed206f92..b6c0aed6e 100644
 --- a/NuGet.Config
 +++ b/NuGet.Config
-@@ -7,7 +7,5 @@
+@@ -7,7 +7,6 @@
      <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
      <add key="dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
      <add key="dotnet-roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
 -    <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
      <add key="dotnet-msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
--    <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
```

Comment:
This feed is needed now? (Suggests a prebuilt given the name.)

Reply:
Nope, just messed up the fixup.  Fixed.